test/CMakeLists.txt

```diff
@@ -56,10 +56,6 @@ set(test_SRCS
  		spells/targetConditions/ReceptiveFeatureConditionTest.cpp
  		spells/targetConditions/SpellEffectConditionTest.cpp
  		spells/targetConditions/TargetConditionItemFixture.cpp
-		
-		vcai/mock_ResourceManager.cpp
```

Comment:
Need to find a way to test VCAI and other dynamic libs without tons of DLL_LINKAGE

Reply:
Anyway tests does not run for now. They require VLC and HOMM3 data